commands/hset.md

```diff
@@ -2,12 +2,11 @@ Sets `field` in the hash stored at `key` to `value`.
 If `key` does not exist, a new key holding a hash is created.
 If `field` already exists in the hash, it is overwritten.
 
-@return
+As of Redis 4.0.0, HSET is variadic and allows for multiple `field`/`value` pairs.
 
-@integer-reply, specifically:
+@return
 
-* `1` if `field` is a new field in the hash and `value` was set.
-* `0` if `field` already exists in the hash and the value was updated.
+@integer-reply: The number of fields that were added or updated.
```

Comment:
"or updated" is incorrect empirically and from a cursory code examination - is there a citable source?